src/Sulu/Bundle/SecurityBundle/Tests/Unit/SingleSignOn/Adapter/OpenId/OpenIdSingleSignOnAdapterTest.php

```diff
@@ -180,20 +184,54 @@ function($method, $url, $options): MockResponse {
             ->willReturn('https://sulu.io/admin');
 
         $this->userRepository->findOneBy(Argument::any())->willReturn(null);
-        $this->contactRepository->createNew()->willReturn($this->prophesize(Contact::class)->reveal());
-        $this->entityManager->persist(Argument::any())->shouldBeCalled();
-        $role = $this->prophesize(Role::class);
```

Comment:
removed the mocks we should never use mocks for entities, its unnecessary overhead